project/package.scala

```diff
@@ -0,0 +1,7 @@
+package object akka {
+
+  implicit class RichBoolean(val b: Boolean) extends AnyVal {
+    final def apply[A](a: => A): Option[A] = if (b) Some(a) else None
+  }
```

Comment:
I would prefer to avoid such very broad conversions (on primitive type, providing an apply()), or if really needed define very near it's use-site.